src/content/learn/passing-props-to-a-component.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Passing Props to a Component
+title: Mengirim Props Pada Komponen
```

Comment:
`pada` or `kepada`?

Because the source is `to`, not `on`. Also in Indonesian, either `pada` or `kepada` will always be in lowercase when being used in a title.


```suggestion
title: Mengirim Props kepada Komponen
```

Reply:
`Pass` lebih dekat dengan `mengoper` layaknya di sepakbola.

`Mengoper Props ke Komponen` should be enough

Reply:
Alright then. I'll change it right away. Thank for the review!

Reply:
Alright, `Mengoper Props ke Komponen`